quickstart/201-mysql-fs-db/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "name_prefix" {
```

Comment:
We should add the expected type to these variables as well.

Reply:
Updated